pkg/util/log/format_crdb.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2021 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package log
+
+import (
+	"io"
+
+	"github.com/cockroachdb/cockroach/pkg/util/log/logpb"
+)
+
+const severityChar = "IWEF"
+
+// MessageTimeFormat is the format of the timestamp in log message headers of crdb formatted logs.
+// as used in time.Parse and time.Format.
+const MessageTimeFormat = "060102 15:04:05.999999"
+
+// FormatLegacyEntry writes the contents of the legacy log entry struct to the specified writer.
+func FormatLegacyEntry(e logpb.Entry, w io.Writer) error {
```

Comment:
Doesn't need to be in this PR but how would you feel about making Format into a type. It can have methods like FormatEntry and NewDecoder.

Reply:
Sounds like a good idea to me!